src/algorithms/tracking/TrackPropagation.cc

```diff
@@ -290,8 +290,10 @@ namespace eicrecon {
         m_log->trace("    loc err = {:.4f}", static_cast<float>(covariance(Acts::eBoundLoc1, Acts::eBoundLoc1)));
         m_log->trace("    loc err = {:.4f}", static_cast<float>(covariance(Acts::eBoundLoc0, Acts::eBoundLoc1)));
 
+	uint16_t detectorID = 0; // default value...will be set in TrackPropagation factory
+        uint16_t surfaceID = targetSurf->geometryId().value();
```

Comment:
Not clear to me where the uint16_t comes from.
```suggestion
	uint8_t system = 0; // default value...will be set in TrackPropagation factory
        uint64_t surface = targetSurf->geometryId().value();
```

Reply:
This was an option proposed in last week's S&C meeting.  As discussed in the meeting I summarized the proposed solutions in slides and gave Daniel's and my recommendations for how to move forward (see discussion [here](https://github.com/eic/EDM4eic/pull/41).  Nobody gave any comments or input so I proceeded with our recommendation.

Reply:
Alignment doesn't matter: fields can be re-ordered to remove padding, unaligned access has no performance penalty on any applicable CPU.

Reply:
I'm not sure why this feedback is coming now on this PR, instead of two weeks ago on the EDM4eic PR?  It was clearly stated in the summary of the S&C meeting that I would proceed with uint16_t unless others preferred something else.

Anyways, do you prefer uint64_t for the surface field?  Please let me know ASAP so that I can change this and the EDM4eic PR.